### PR TITLE
Fixes #4

### LINKED ISSUE #4: Serialization exception when using Container Managed Security (Tomcat Realms)
Environment:
Tomcat 8.0.20
hazelcast-all-3.6.4
hazelcast-tomcat8-sessionmanager-1.0
with „Container Managed Security” (Apache Tomcat Realm)

```
Caused by: java.io.NotSerializableException: org.apache.catalina.authenticator.SavedRequest
      at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1184)
      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:348)
      at java.util.Hashtable.writeObject(Hashtable.java:1157)
      at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
      at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
      at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
      at java.lang.reflect.Method.invoke(Method.java:498)
      at java.io.ObjectStreamClass.invokeWriteObject(ObjectStreamClass.java:1028)
      at java.io.ObjectOutputStream.writeSerialData(ObjectOutputStream.java:1496)
      at java.io.ObjectOutputStream.writeOrdinaryObject(ObjectOutputStream.java:1432)
      at java.io.ObjectOutputStream.writeObject0(ObjectOutputStream.java:1178)
      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:348)
      at com.hazelcast.internal.serialization.impl.JavaDefaultSerializers$JavaSerializer.write(JavaDefaultSerializers.java:242)
      at com.hazelcast.internal.serialization.impl.StreamSerializerAdapter.write(StreamSerializerAdapter.java:41)
      at com.hazelcast.internal.serialization.impl.AbstractSerializationService.writeObject(AbstractSerializationService.java:197)
      ... 49 more
```

### PR DESCRIPTION
**Pull request by emre-aydin**

### CODE REVIEW

**Comment by mesutcelik** [comment]
run-lab-run

**Comment by mesutcelik** [comment]
verify

File: tomcat6/src/test/java/com/hazelcast/session/Tomcat6SecurityTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import com.hazelcast.core.Hazelcast;
+import org.apache.http.client.CookieStore;
+import org.apache.http.impl.client.BasicCookieStore;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class Tomcat6SecurityTest extends AbstractHazelcastSessionsTest {
```
Comment (mesutcelik) [inline tomcat6/src/test/java/com/hazelcast/session/Tomcat6SecurityTest.java]: missing test category?

File: tomcat6/src/test/java/com/hazelcast/session/Tomcat6Configurator.java [hunk 2]
```diff
@@ -18,40 +20,38 @@
...
     private static String DEFAULT_HOST = "localhost";
 
-    private String p2pConfigLocation;
-    private String clientServerConfigLocation;
+    private String appName;
 
-    public Tomcat6Configurator(String p2pConfigLocation, String clientServerConfigLocation) {
-        super();
-        this.p2pConfigLocation = p2pConfigLocation;
-        this.clientServerConfigLocation = clientServerConfigLocation;
```
Comment (mesutcelik) [inline tomcat6/src/test/java/com/hazelcast/session/Tomcat6Configurator.java]: so you think those are not needed?

**Comment by mesutcelik** [comment]
@emre-aydin ,
can you please check if tests have been executed without `Category`?

if so then 👍 with one minor comment

File: tomcat6/src/test/java/com/hazelcast/session/Tomcat6Configurator.java (hunk 2 above)
Comment (emre-aydin) [inline tomcat6/src/test/java/com/hazelcast/session/Tomcat6Configurator.java]: yep, they are unused.

**Comment by emre-aydin** [comment]
they had been executed, added `Category` nonetheless

**Comment by mesutcelik** [comment]
Thx @emre-aydin ,

I think the same fix can be applied to #2 as well. Can you please send another PR for it?

**Pull request merged**